helsingborg-stad/app-mitt-helsingborg
# Feat/symmetric encryption

## Explain the changes you’ve made
Added functionality for encrypting and decrypting with a symmetric key.

## Explain why these changes are made
This is a necessary part of the encryption workflow for co-applicants.

## Explain your solution
Added `encryptWithSymmetricKey` and `decryptWithSymmetricKey` following the same pattern as the existing setup for non-symmetric encryption, and updated `encryptFormAnswers` and `decryptFormAnswers` to use symmetric encryption if the form should use it.

Also updated the symmetric key demo in the encryption storybook.

## How to test the changes?
1) Checkout this branch
2) Run `yarn test` and check that all encryption tests pass
3) Switch to Storybook
4) Run the symmetric key demo in "EncryptionService / terminal demo"
5) Verify that the encryption and decryption succeeds and that both applicant and co-applicant decrypts to the same answers object

## Was this feature tested in the following environments?
- [] Building the Application on a Android device/simulator.
